### PR TITLE
Typo fix in voyager/README.md

### PR DESCRIPTION
**Pull request by AdamDang**
Line 84: Delete duplicated "the the"

### CODE REVIEW

**Comment by tamalsaha** [comment]
@timstoop, some validation error in values.yaml . Do you know what is the issue?

**Comment by timstoop** [comment]
Yes, this is a recent change in the CI system, now the home key in Chart.yaml is required. If you feel like it, please add it. That will solve it for all future commits to this Chart. Otherwise you'll have to wait until someone else adds it.

**Comment by AdamDang** [comment]
@amalsaha  @timstoop 
I've add the home key for the Chart.yaml.  It seems OK now.

File: stable/voyager/Chart.yaml [hunk 1]
```diff
@@ -1,8 +1,9 @@
 apiVersion: v1
 description: 'Voyager by AppsCode - Secure Ingress Controller for Kubernetes'
+home: https://cdn.appscode.com/
```
Comment (tamalsaha) [inline stable/voyager/Chart.yaml]: Please set this to `https://appscode.com/products/voyager/`

Comment (AdamDang) [reply stable/voyager/Chart.yaml]: OK, it's done.

**Pull request merged**